packages/rocketchat-setup-wizard/server/lib/getWizardSettings.js

```diff
@@ -0,0 +1,9 @@
+Meteor.methods({
+	getWizardSettings() {
```

Comment:
this should be protected somehow.. currently it is returning all `wizard` settings to everyone, including its values, which may contain sensitive data.

since the wizard will appear to the first user only, I suggest checking the same here.. so, only return `wizard` settings if there is no admin set.

Reply:
This should return only for the admins, and the user creation should happen before the other wizard steps